sdk/apicenter/Azure.ResourceManager.ApiCenter/assets.json

```diff
@@ -1,6 +1,6 @@
 {
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
-  "TagPrefix": "net/<service name>/Azure.ResourceManager.ApiCenter",
+  "TagPrefix": "net/apicenter/Azure.ResourceManager.ApiCenter",
```

Comment:
Since this is stable release please follow
[azure-sdk-for-net/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md at main · Azure/azure-sdk-for-net (github.com)](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md)
to add tests and record the test result